### ISSUE TITLE
After applying resize, menu items are not accessible using keyboard: A11y_Aspire Dashboard_Keyboard.

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/aspire#10139]
#A11ySev2; #A11yTCS;#DesktopWeb; #Win32;#.NET Project system for CSharp and VB;#BM_AspireDashboard_Web_June25;#Win32;#WCAG2.1.1;#Keyboard;#A11yMAS;

### Environment Details:
Application Name: Aspire Dashboard
Edition Windows 11 Enterprise
Version 24H2
OS build 26100.4351

Pre-Requisites:
Go to Settings > System > Display
To ensure consistent testing, set the machines target resolution to an "average" mid-range resolution (1280 x 768). And scaling should be at 100%.
Zoom the page to 200% and refresh the page.

### Repro Steps

1. Open Visual studio.
2. Open a aspire project and run it.
3. A new window will open in the chromium edge,
4. Verify that After applying resize, menu items are not accessible using keyboard.

### Actual Result:
After applying resize, menu items are not accessible using keyboard.

### Expected Result:
After resizing, the menu items should remain accessible using the keyboard.

### User Impact:
Low vision and Keyboard users will face difficulty as menu items are not accessible after applying resize resolution.

### Attachment:

https://github.com/user-attachments/assets/626de424-6455-41d4-8670-6c1e3f4b9696

**Comment by adamint** [comment]
Fixed in next fluentui release
